packages/slate-hyperscript/test/decorations/across-blocks.js

```diff
@@ -0,0 +1,87 @@
+/** @jsx h */
+
+import { createHyperscript } from '../..'
+
+const h = createHyperscript({
+  blocks: {
+    paragraph: 'paragraph',
+  },
+  decorators: {
+    highlight: 'highlight',
+  },
+})
+
+export const input = (
+  <value>
+    <document>
+      <paragraph>
+        This is one <highlightAnchor key="a" />block.
```

Comment:
What are the decoration `key` used for? 
Cannot you use the decoration `key` as a way to pair the start/end of the decoration?

If not, I would suggest the following syntax:

```jsx
<highlight anchor key="a"/>
<highlight focus key="a" />
```